src/datasets/tasks/automatic_speech_recognition.py

```diff
@@ -0,0 +1,18 @@
+from dataclasses import dataclass
+from typing import ClassVar, Dict
+
+from ..features import Features, Value
+from .base import TaskTemplate
+
+
+@dataclass(frozen=True)
+class AutomaticSpeechRecognition(TaskTemplate):
+    task: str = "automatic-speech-recognition"
+    input_schema: ClassVar[Features] = Features({"audio_file": Value("string")})
```

Comment:
Note that in the feature we'll have an Audio feature type that will probably have additional parameters (like ClassLabel) such as the sampling rate or the audio format.

Reply:
good to know!